pontoon/settings/test.py

```diff
@@ -1,3 +1,4 @@
 """Django settings for the test suite."""
 
 PIPELINE_ENABLED = False
+CELERY_ALWAYS_EAGER = 1
```

Comment:
Ah, sorry - i'll fix it.

On Wed, Nov 11, 2015 at 7:42 PM, Michael Kelly <notifications@github.com>
wrote:

> In pontoon/settings/test.py
> <https://github.com/mozilla/pontoon/pull/248#discussion_r44568595>:
>
> > @@ -1,3 +1,4 @@
> >  """Django settings for the test suite."""
> >
> >  PIPELINE_ENABLED = False
> > +CELERY_ALWAYS_EAGER = 1
>
> True works fine as well. :P
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/pontoon/pull/248/files#r44568595>.
>



-- 
jarek@reijutsu:~$ fortune
You have Egyptian flu: you're going to be a mummy.
jarek@reijutsu:~$ fortune
You now have Asian Flu.